src/aiortc/codecs/keypoints_pb2.py

```diff
@@ -18,7 +18,8 @@
   package='keypoint',
   syntax='proto2',
   serialized_options=None,
-  serialized_pb=b'\n\x0fkeypoints.proto\x12\x08keypoint\"&\n\x08Keypoint\x12\x0c\n\x04xloc\x18\x01 \x02(\x02\x12\x0c\n\x04yloc\x18\x02 \x02(\x02\">\n\x08Jacobian\x12\x0b\n\x03\x64\x31\x31\x18\x01 \x02(\x02\x12\x0b\n\x03\x64\x31\x32\x18\x02 \x02(\x02\x12\x0b\n\x03\x64\x32\x31\x18\x03 \x02(\x02\x12\x0b\n\x03\x64\x32\x32\x18\x04 \x02(\x02\"x\n\x0cKeypointInfo\x12%\n\tkeypoints\x18\x01 \x03(\x0b\x32\x12.keypoint.Keypoint\x12%\n\tjacobians\x18\x02 \x03(\x0b\x32\x12.keypoint.Jacobian\x12\x0b\n\x03pts\x18\x03 \x02(\x02\x12\r\n\x05index\x18\x04 \x02(\x05'
+  create_key=_descriptor._internal_create_key,
+  serialized_pb=b'\n\x0fkeypoints.proto\x12\x08keypoint\"&\n\x08Keypoint\x12\x0c\n\x04xloc\x18\x01 \x02(\x02\x12\x0c\n\x04yloc\x18\x02 \x02(\x02\">\n\x08Jacobian\x12\x0b\n\x03\x64\x31\x31\x18\x01 \x02(\x02\x12\x0b\n\x03\x64\x31\x32\x18\x02 \x02(\x02\x12\x0b\n\x03\x64\x32\x31\x18\x03 \x02(\x02\x12\x0b\n\x03\x64\x32\x32\x18\x04 \x02(\x02\"\x94\x01\n\x0cKeypointInfo\x12%\n\tkeypoints\x18\x01 \x03(\x0b\x32\x12.keypoint.Keypoint\x12%\n\tjacobians\x18\x02 \x03(\x0b\x32\x12.keypoint.Jacobian\x12\x0b\n\x03pts\x18\x03 \x02(\x02\x12\x13\n\x0b\x66rame_index\x18\x04 \x02(\x05\x12\x14\n\x0csource_index\x18\x05 \x02(\x05'
```

Comment:
how to you get this string? it's `rame_index`, should it be `frame_index` or does it get confused by `0xf`?

Reply:
generated file - you're reading hex values, don't forget.